File: app/src/main/java/ir/ahfz/rentcar/io/network/model/BranchResponse.java
package ir.ahfz.rentcar.io.network.model;

import java.util.List;

import ir.ahfz.rentcar.io.network.BaseResponse;

public class BranchResponse extends BaseResponse {

    private List<Branch> branches = null;

    public List<Branch> getBranches() {
        return branches;
    }

    public void setBranches(List<Branch> branches) {
        this.branches = branches;
    }


    public static class Branch {

        private String id;
        private String name;
        private String email;
        private String address;
        private String cityId;
        private String phone;
        private String createdAt;
        private String updatedAt;
        private String city;

        public String getId() {
            return id;
        }

        public void setId(String id) {
            this.id = id;
        }

        public String getName() {
            return name;
        }

        public void setName(String name) {
            this.name = name;
        }

        public String getEmail() {
            return email;
        }

        public void setEmail(String email) {
            this.email = email;
        }

        public String getAddress() {
            return address;
        }

        public void setAddress(String address) {
            this.address = address;
        }

        public String getCityId() {
            return cityId;
        }

        public void setCityId(String cityId) {
            this.cityId = cityId;
        }

        public String getPhone() {
            return phone;
        }

        public void setPhone(String phone) {
            this.phone = phone;
        }

        public String getCreatedAt() {
            return createdAt;
        }

        public void setCreatedAt(String createdAt) {
            this.createdAt = createdAt;
        }

        public String getUpdatedAt() {
            return updatedAt;
        }

        public void setUpdatedAt(String updatedAt) {
            this.updatedAt = updatedAt;
        }

        public String getCity() {
            return city;
        }

        public void setCity(String city) {
            this.city = city;
        }

    }
}

File: app/src/main/java/ir/ahfz/rentcar/io/network/model/BranchResponse.kt
package ir.ahfz.rentcar.io.network.model

import ir.ahfz.rentcar.io.network.BaseResponse

/**
 * Data model for Branch response
 */
data class BranchResponse(var branches: List<Branch>? = null) : BaseResponse() {

    /**
     * Data model for Branch
     */
    data class Branch(
        var id: String? = null,
        var name: String? = null,
        var email: String? = null,
        var address: String? = null,
        var cityId: String? = null,
        var phone: String? = null,
        var createdAt: String? = null,
        var updatedAt: String? = null,
        var city: String? = null
    )
}
